### ISSUE TITLE
semi-supervised_learning typo

### ISSUE THREAD

**Issue by sridhar912** [udacity/dlnd-issue-reports#317]
https://github.com/udacity/deep-learning/tree/master/semi-supervised

In both quiz and solution file, there is typo in discriminator function. 

x7 = tf.layers.conv2d(relu5, 2 * size_mult, 3, strides=1, padding='valid')

Here the relu5 should be actually relu6
